testsuite/tests/language-extensions/language_extensions.ml

```diff
@@ -2,118 +2,148 @@
    include ocamlcommon
    flags = "-I ${ocamlsrcdir}/parsing"
    reference = "${test_source_directory}/reference.txt"
-   * skip
-   reason = "local_ currently always parses"
 *)
 
+(* Currently, this test case only works with modular extensions, as they fail
```

Comment:
This comment seems at odd with your GitHub comment which said that modular extensions *don't* fail during parsing...

Reply:
Yes, that's because I caught that during rebase and punted on the best way to resolve it